src/subgraph/types.ts

```diff
@@ -0,0 +1,26 @@
+export interface DepositDto {
+  id: string;
+  by: string;
+  depositId: string;
+  amount: string;
+  lockedFrom: string;
+  lockedUntil: string;
+  pool: string;
+  timestamp: string;
+}
+
+export interface DepositsDto {
+  deposits: DepositDto[];
+}
+
+export interface RewardDto {
+  id: string;
+  for: string;
+  amount: string;
+  pool: string;
```

Comment:
This is still not correct.

Reply:
When you do a graphql query

```
rewards {
  pool {
    id
  }
}
```

You do not get back a `string` for `pool`.

Please console log what the output actually is like I mentioned earlier so you can see what I am talking about.